lib/ood_packaging/build_box/docker-image/Dockerfile.erb

```diff
@@ -32,7 +32,7 @@ RUN apt update -y && apt install -y apt-transport-https ca-certificates \
     init debhelper devscripts dh-make build-essential apt-cudf lintian equivs \
     sudo rake wget curl ruby bundler && \
     apt clean all -y
-<% if ! ['bookworm'].include?(codename) && arch != 'ppc64le' %>
+<% if ! ['noble', 'bookworm'].include?(codename) && arch != 'ppc64le' %>
```

Comment:
Seems like it's more future resistant to just have `codename == 'bookworm'`? I mean won't this break/need updating when `26.04` comes out?

Reply:
Errr sorry, I'm getting turned around `codename == jammy`? I can't recall the 22 & 20 codenames, but it seems like it should check for those so that future releases won't need this update.